Release notes/Release.md

```diff
@@ -0,0 +1,243 @@
+## 22 March 2023
+
+<!DOCTYPE html>
+<html>
+<body>
+<table>
```

Comment:
ok added KC, also removed the <html>
<body> as well otherwise alignment not ok.
verified in intra > qa , https://docs.developer.tech.gov.sg/docs/qa-adex-user-guide-for-intranet/Release%20notes/Release?id=_18-january-2023
Thanks.